### PR TITLE
Latest pl_PL for 3.x - pkp-lib

### PR DESCRIPTION
**Pull request by szmigieldesign**
Source: http://szmigiel.design/files/ojs-31-pl_PL-19-03-2018.tar.gz

### CODE REVIEW

File: plugins/metadata/nlm30/locale/pl_PL/locale.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	<message key="plugins.metadata.nlm30.citationOutput.displayName">Citation Output Services</message>
+	<message key="plugins.metadata.nlm30.citationOutput.description">Konwertuj dokument ze znacznikami cytowań NLM 3.0 do standardowego tekstu.</message>
+	<message key="plugins.metadata.nlm30Xml.citationOutput.displayName">Citation Output Services (XML)</message>
+	<message key="plugins.metadata.nlm30Xml.citationOutput.description">Konwertuj dokument ze znacznikami cytowań NLM 3.0 do standardowego pliku XML z cytowaniami.</message>
+	<message key="plugins.metadata.nlm30.nlm30XmlOutput.displayName">NLM 3.0 Journal Publishing Output</message>
+	<message key="plugins.metadata.nlm30.nlm30XmlOutput.description">Konwertuj tekst do NLM 3.0.</message>
+	<message key="plugins.metadata.nlm30.nlm23XmlOutput.displayName">NLM 3.0 Journal Publishing Output</message>
+	<message key="plugins.metadata.nlm30.nlm23XmlOutput.description">Konwertuj tekst do NLM 2.3.</message>
+	<message key="plugins.metadata.nlm30.nlm30ToNlm23Xml.displayName">NLM 3.0 do 2.3 Journal Publishing Downgrade</message>
+	<message key="plugins.metadata.nlm30.nlm30ToNlm23Xml.description">Downgrades NLM 3.0 do NLM 2.3.</message>
```
Comment (mtub) [inline plugins/metadata/nlm30/locale/pl_PL/locale.xml]: English text…

File: plugins/generic/usageStats/locale/pl_PL/locale.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	<message key="plugins.generic.usageStats.optout.done"><![CDATA[<p>Zrezygnowałeś/aś z gromadzenie statystyk użycia. Od tego momentu żadne dane o statystykach użycia strony nie będą gromadzone. Kliknij przycisk poniżej, aby zmienić swoją decyzję.</p>]]></message>
+	<message key="plugins.generic.usageStats.optin">Włączenie</message>
+	<message key="plugins.generic.usageStats.optout">Rezygnacja</message>
+	<message key="plugins.generic.usageStats.optout.cookie"><![CDATA[<p>If you wish you can opt-out of the data collection process. By clicking the opt-out button below, you can actively decide against participating in the statistical analysis. When clicking the opt-out button a <em>cookie</em> is being created on your system to store your decision. If the privacy settings of your browser lead to cookies being automatically deleted you will have to opt-out again the next time you access this website. The cookie is only valid for one browser. If you use a different browser, you will have to opt out again. No individual information is stored within this cookie. This cookie lease is valid for one year after your last access.</p>
```
Comment (mtub) [inline plugins/generic/usageStats/locale/pl_PL/locale.xml]: English text

**Comment by szmigieldesign** [comment]
Fixed, shall I request another pull?

**Comment by mtub** [comment]
No need for another pull request, GitHub updates the pull request automatically when you send new commits to your repository.

**Comment by mtub** [comment]
This looks good, many thanks for the update, @szmigieldesign !

**Pull request merged**